Packs/GreatHorn/Integration/GreatHorn/GreatHorn.yml

```diff
@@ -0,0 +1,868 @@
+    - contextPath: GreatHorn.Message.headers.x-tm-id
+      description: ""
+      type: Unknown
+    - contextPath: GreatHorn.Message.headers.x-rpcampaign
+      description: ""
+      type: Unknown
+    - contextPath: GreatHorn.Message.headers.x-google-smtp-source
+      description: ""
+      type: Unknown
+    - contextPath: GreatHorn.Message.headers.content-type
+      description: ""
+      type: Unknown
+  - name: gh-remediate-message
+    description: Perform the specified remediation action on message
+    arguments:
+    - name: action
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - archive
+      - banner
+      - delete
+      - label
+      - move
+      - quarantine
+      - delete
+      - removeattachments
+      - review
+      - trash
+      description: The action to take on the given message
+    - name: eventId
+      required: true
+      description: The GreatHorn event ID
+    - name: hasButton
+      auto: PREDEFINED
+      predefined:
+      - "True"
+      - "False"
+      description: If true, the banner will include a button enabling the end-user
+        to remove the banner. Default is True.
+      defaultValue: "True"
+    - name: message
+      description: The text to display in the email's banner.
+    - name: label
+      description: The name of the label to add. If the label name does not exist,
+        it will be created.
+    - name: location
+      description: The target location in the user's mailbox. If the location does
+        not exist, it will be created.
+    outputs:
+    - contextPath: GreatHorn.Remediation.action
+      description: ""
+      type: String
+    - contextPath: GreatHorn.Remediation.eventId
+      description: ""
+      type: String
+    - contextPath: GreatHorn.Remediation.reason
+      description: ""
+      type: String
+    - contextPath: GreatHorn.Remediation.success
+      description: ""
+      type: Number
+  - name: gh-revert-remediate-message
+    description: Revert the specified remediation action on the given message
+    arguments:
+    - name: action
+      description: Remediation action to revert
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - banner
+      - quarantinerequest
+      - quarantinerelease
+      - quarantinedeny
+      - removeattachments
+      - review
+    - name: eventId
+      required: true
+      description: The GreatHorn event ID
+    outputs:
+    - contextPath: GreatHorn.Remediation.action
+      description: ""
+      type: String
+    - contextPath: GreatHorn.Remediation.eventId
+      description: ""
+      type: String
+    - contextPath: GreatHorn.Remediation.reason
+      description: ""
+      type: String
+    - contextPath: GreatHorn.Remediation.success
+      description: ""
+      type: Number
+  - name: gh-get-policy
+    description: Retrieve details about the policy specified
+    arguments:
+    - name: policyid
+      description: The GreatHorn policy ID 
```

Comment:
Please note: If a command uses argToList function the argument `isArray` should be set to true.

Reply:
done